apps/dashboard/app/controllers/projects_controller.rb

```diff
@@ -26,8 +26,9 @@ def edit
   def update
     @project = Project.find(params[:id])
 
-    if @project.update(project_params)
-      @project.write_manifeset
+    if @project.save!
+      @project.update(project_params)
```

Comment:
```suggestion
    if @project.update(project_params)
```

Why save and update?